hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/state/ProxyWorldUpdater.java

```diff
@@ -424,6 +426,24 @@ public void externalizeSystemContractResults(
         enhancement.systemOperations().externalizeResult(result, status);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @NonNull
+    public ExchangeRate currentExchangeRate() {
+        return enhancement().systemOperations().currentExchangeRate();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @NonNull
+    public ContractsConfig contractsConfig() {
```

Comment:
The operation's `ContractsConfig` is already set in the frame's context variables (helper to access it [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/utils/FrameUtils.java#L42))...did you add here to account for a future possibility of the config changing mid-operation?

Reply:
Didn't realize that was there.  Thanks for pointing that out.